openlibrary/admin/templates/admin/index.html

```diff
@@ -178,12 +186,12 @@ <h2>Members</h2>
         <div class="titleSmall fixthis">Prolific Member Editors</div>
         <div class="smaller">
             <a href="[USER PROFILE]">name</a>,
-            <a href="[USER PROFILE"]>name</a>,
-            <a href="[USER PROFILE"]>name</a>,
-            <a href="[USER PROFILE"]>name</a>,
-            <a href="[USER PROFILE"]>ip address</a>,
-            <a href="[USER PROFILE"]>name</a>,
-            <a href="[USER PROFILE"]>name</a>
+            <a href="[USER PROFILE" ]>name</a>,
```

Comment:
Looks like you added some additional spaces here

Reply:
And changed how this functions?

Is it
href="[USER PROFILE]"
Or
href="[USER PROFILE"] ?

I'm guessing all of these should be consistent.